src/components/textfield/TextField.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './textfield.scss';
+
+class TextField extends React.Component {
+  render() {
+    const { value, onChange, onFocus, onBlur, label, placeholder, type, multiline } = this.props;
+
+    const _renderTextInput = () => {
```

Comment:
I don't see the need of creating an extra function here, please remove it

Reply:
+1

Reply:
in order to render textarea when multiline= true, how can i do conditional rendering?

Reply:
Good question!

**This is an approach:**
```
class TextField extends React.Component {
  render() {
    const { value, onChange, onFocus, onBlur, label, placeholder, type, multiline } = this.props;

    return (
      <div className={styles.root}>
        <label className={styles.labelStyle}>{label}</label>
        <input
          className={multiline ? styles.textAreaStyle : styles.inputStyle}
          label={label}
          onChange={onChange}
          onBlur={onBlur}
          onFocus={onFocus}
          type={type}
          value={value}
          placeholder={placeholder}
        />
      </div>
    );
  }
}
```

**In my opinion this other approach looks better though:** 
```
const TextField = ({
  value,
  onChange,
  onFocus,
  onBlur,
  label,
  placeholder,
  type,
  multiline
}) => (
  <div className={styles.root}>
      <label className={styles.labelStyle}>{label}</label>
      <input
        className={multiline ? styles.textAreaStyle : styles.inputStyle}
        label={label}
        onChange={onChange}
        onBlur={onBlur}
        onFocus={onFocus}
        type={type}
        value={value}
        placeholder={placeholder}
      />
    </div>
)
```